guides/bulk.md

```diff
@@ -0,0 +1,271 @@
+        json!({ "index": { "_index": movies, "_id": "1" }}).into(),
+        json!({ "title": "Beauty and the Beast", "year": "1991" }).into(),
+        json!({ "index": { "_index": movies, "_id": "2" }}).into(),
+        json!({ "title": "Beauty and the Beast - Live Action", "year": "2017" }).into(),
+        json!({ "index": { "_index": books, "_id": "1" }}).into(),
+        json!({ "title": "The Lion King", "year": "1994" }).into(),
+    ]))
+    .send()
+    .await?;
+```
+
+As you can see, each bulk operation is comprised of two objects. The first object contains the operation type and the target document's `_index` and `_id`. The second object contains the document's data. As a result, the body of the request above contains six objects for three index actions.
+
+Alternatively, the `bulk` method can accept an array of hashes where each hash represents a single operation. The following code is equivalent to the previous example:
+
+```rust
+let mut ops = BulkOperations::new();
+ops.push(
+    BulkOperation::index(json!({
+        "title": "Beauty and the Beast",
+        "year": "1991"
+    }))
+    .index(movies)
+    .id("1"),
+)?;
+ops.push(
+    BulkOperation::index(json!({
+        "title": "Beauty and the Beast - Live Action",
+        "year": "2017"
+    }))
+    .index(movies)
+    .id("2"),
+)?;
+ops.push(
+    BulkOperation::index(json!({
+        "title": "The Lion King",
+        "year": "1994"
+    }))
+    .index(books)
+    .id("1"),
+)?;
+
+client.bulk(BulkParts::None).body(vec![ops]).send().await?;
+```
+
+We will use this format for the rest of the examples in this guide.
+
+### Creating multiple documents
+
+Similarly, instead of calling the `create` method for each document, you can use the `bulk` API to create multiple documents in a single request. The following code creates three documents in the `movies` index and one in the `books` index:
+
+```rust
+let mut ops = BulkOperations::new();
+ops.push(
+    BulkOperation::create(json!({
+        "title": "Beauty and the Beast 2",
+        "year": "2030"
+    }))
+    .index(movies),
+)?;
+ops.push(
+    BulkOperation::create(json!({
+        "title": "Beauty and the Beast 3",
+        "year": "2031"
+    }))
+    .index(movies),
+)?;
+ops.push(
+    BulkOperation::create(json!({
+        "title": "Beauty and the Beast 4",
+        "year": "2049"
+    }))
+    .index(movies),
+)?;
+ops.push(
+    BulkOperation::create(json!({
+        "title": "The Lion King 2",
+        "year": "1998"
+    }))
+    .index(books),
+)?;
+client.bulk(BulkParts::None).body(vec![ops]).send().await?;
+```
+
+Note that we specified only the `_index` for the last document in the request body. This is because the `bulk` method accepts an `index` parameter that specifies the default `_index` for all bulk operations in the request body. Moreover, we omit the `_id` for each document and let OpenSearch generate them for us in this example, just like we can with the `create` method.
```

Comment:
```suggestion
Note that we specified only the `_index` for the last document in the request body. This is because the `bulk` method accepts an `index` parameter that specifies the default `_index` for all bulk operations in the request body. Moreover, we omit the `_id` for each document and let OpenSearch generate them for us in this example, just like we can with the `index` method.
```